lib/tasks/export/analytics_bad_links_exporter.rake

```diff
@@ -7,7 +7,7 @@ namespace :export do
       service_desc = "Export bad links status to Google Analytics"
       begin
         Rails.logger.info("Starting link exporter")
-        Services.icinga_check(service_desc, true, "Exporting bad links list to Google Analytics")
+        Services.icinga_check(service_desc, true, "Exporting list of bad links to Google Analytics")
```

Comment:
Nitpick, but this change could be squashed down into edbd040b364e182878eb3976e56c71142c2d6bdf instead of living in this commit.